CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ This project attempts to follow [semantic versioning](https://semver.org/)
   * Not working on OSX - macs don't read from /etc/profile.d/
   * Stops showing color if you `sudo su`
 
+## 1.0.9
+
+* enhancements
+  * Add bundler_version to install specific bundler version. Is required.
```

Comment:
If this is required, then this is a breaking change, right? So major version bump? Or make it not required somehow (maybe default to the last 1.x.x version?).

Reply:
Yeah I never know how to handle this. If i default it to 1 then all the ones that need 2 will break if they update. I kind of like it just throwing an error if you haven't set it. I actually kind of want subspace to not have any defaults for anything.

Reply:
Yea throwing an error is fine, but I think you have to do a major version bump to make that change because it will break for anyone that just upgrades to 1.0.9 without doing anything

Reply:
Oh yeah that's fine I can do that. v2 it is